cxx-qt-gen/src/parser/mod.rs

```diff
@@ -0,0 +1,166 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+pub mod qobject;
+
+use qobject::{find_and_merge_cxx_qt_item, find_qobjects, ParserCxxQtData};
+use syn::{token::Brace, ItemMod, Result};
+
+/// A struct representing a module block with CXX-Qt relevant Items into ParserCxxQtData
+pub struct Parser {
+    /// The module which unknown (eg CXX) blocks are stored into
+    pub module: ItemMod,
```

Comment:
When I first saw this variable name I presumed it was for the cxx-qt module, but the comment says it's the opposite... can you think of a more descriptive name than simply `module`? Something like `non_cxx_qt_module`, but less wordy? `passthrough_module`? `foreign_module`?

Reply:
Right, so if you had like this

```
#[cxx_qt::bridge]
mod ffi {
  extern "Rust" {
    fn test();
  }

  #[cxx_qt::qobject]
  struct MyObject;
}
```

The split that occurs here is the following...

### module

Gets the original module data, and any non CXX-Qt items, note it potentially still has the cxx_qt::bridge attribute (depending how earlier macro phases work). This is because the parser phase is simply annotating the structure rather than rewriting syn items. Later the generate Rust side will change that to `#[cxx::bridge]`.
```
#[cxx_qt::bridge]
mod ffi {
  extern "Rust" {
    fn test();
  }
}
```

### cxx_qt_data

Gets any CXX-Qt items and will try to annotate them later with extra info rather than just storing the Item itself.

```
  #[cxx_qt::qobject]
  struct MyObject;
```

So probably `passthrough_module` or `non_annotated` or `insignificant` or something? :thinking:

Reply:
Let's go with `passthrough_module`.